site/en/blog/how-blockbench-uses-the-eyedropper-api/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How the 3D model editor Blockbench uses the EyeDropper API to let users choose colors from everywhere'
+subhead: >
+  The EyeDropper API provides a mechanism for creating an eyedropper tool that lets users sample colors from their screens, including outside of the browser window. Learn how the 3D model editor Blockbench uses this API in their codebase.
+date: 2023-01-31
+# updated: 2023-01-31
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/6fIc5QdWayhjyXmIFEi6.png
+alt: Computer created with Blockbench.
+authors:
+  - thomassteiner
+  - jannispetersen
+tags:
+  - capabilities
+  - case-study
+---
+
+[Blockbench](https://www.blockbench.net/) is a free, modern model editor for low-poly models with pixel art textures. If you have ever played [Minecraft](https://www.minecraft.net/), chances are you have seen assets that were created with Blockbench, for example the goat, which was added to Minecraft as part of the Caves and Cliffs Update, Part 1. The GPL version 3 licensed Blockbench code is [open source](https://github.com/JannisX11/blockbench) and maintained by a friendly community of developers.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/3NK0qVqDGnG7hTpaZpUS.png", alt="Minecraft sheep designed with Blockbench.", width="800", height="426" %}
+
+To try Blockbench, launch the app by navigating to [web.blockbench.net](https://web.blockbench.net/). As a PWA, you can install it to your desktop and launch it in a standalone window. If you are new to 3D modeling, you can start with one of the many [open source `.bbmodel` examples](https://github.com/search?q=path%3A*.bbmodel&type=code) you can find on GitHub (login required). For example, try the train model that you can see in the following screenshot.
```

Comment:
Hmm, I'm not sure why the GitHub search link works for you and doesn't work for me. On my end it just displays this message: 
![image](https://user-images.githubusercontent.com/29278436/215861641-f5eabf6e-5f4b-4cdb-a0d1-cd2c124ed5ec.png)

Reply:
OK, traced down to the new code search experiment. It's incompatible both ways, links to the old version don't work for the new version and vice versa. I will replace the link with text then. 

![image](https://user-images.githubusercontent.com/145676/216041704-447fbfc8-fedb-4217-961e-bdcb9c356ee1.png)